test/functional/src/ec_tests.cpp

```diff
@@ -29,6 +35,11 @@ bool EcTests::IsConfigured()
         return TestFail("keysinuse engine not loaded for EC operations, found [ %s ]", loaded_id);
     }
 
+    return true;
+}
+
+bool EcTests::Setup()
+{
     if (ecBio == nullptr)
```

Comment:
>if (ecBio == nullptr) [](http://example.com/codeflow?start=4&length=21)

Where does this get set?

Reply:
Constructor. I'll move that logic here though.

Reply:
Ah, right. Missed that. I do think it makes sense to initialize it and check the result in the same place.